webapp/components/generic/SearchPost/SearchPost.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <ds-flex class="search-post">
+    <ds-flex-item class="search-option-label">
+      <ds-text>{{ option.title | truncate(70) }}</ds-text>
+    </ds-flex-item>
+    <ds-flex-item class="search-option-meta" width="280px">
+      <ds-flex>
+        <ds-flex-item>
+          <ds-text size="small" color="softer" class="search-post-meta">
+            <span class="comments-count">
+              {{ option.commentsCount }}
+              <base-icon name="comments" />
+            </span>
+            <span class="shouted-count">
+              {{ option.shoutedCount }}
+              <base-icon name="bullhorn" />
+            </span>
+          </ds-text>
+        </ds-flex-item>
+        <ds-flex-item>
+          <ds-text size="small" color="softer" align="right" class="search-post-author">
+            {{ option.author.name | truncate(32) }} -
+            {{ option.createdAt | dateTime('dd.MM.yyyy') }}
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-20T15:55:25Z" title="Friday, December 20th 2019, 4:55:25 pm +01:00">Dec 20, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,66 @@
+<template>
+  <ds-flex class="search-post">
+    <ds-flex-item class="search-option-label">
+      <ds-text>{{ option.title | truncate(70) }}</ds-text>
+    </ds-flex-item>
+    <ds-flex-item class="search-option-meta" width="280px">
+      <ds-flex>
+        <ds-flex-item>
+          <ds-text size="small" color="softer" class="search-meta">
+            <span style="text-align: right;">
+              <b>{{ option.commentsCount }}</b>
+              <base-icon name="comments" />
+            </span>
+            <span class="post-shouted">
+              <b>{{ option.shoutedCount }}</b>
+              <base-icon name="bullhorn" />
+            </span>
+          </ds-text>
+        </ds-flex-item>
+        <ds-flex-item>
+          <ds-text size="small" color="softer" align="right">
+            {{ option.author.name | truncate(32) }} -
+            {{ option.createdAt | dateTime('dd.MM.yyyy') }}
```


Could we localize the datetime here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-20T16:12:38Z" title="Friday, December 20th 2019, 5:12:38 pm +01:00">Dec 20, 2019</time>_

---


maybe